src/main/java/org/fundacionjala/core/util/Environment.java

```diff
@@ -35,7 +35,7 @@ private void readFile() {
             properties.load(input);
         } catch (IOException ex) {
             LOGGER.error(ex.getMessage());
-            //TODO stop execution with message
+            throw new RuntimeException();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)